### ISSUE TITLE
Installation issue on Debian Jessie since version 1.6

### ISSUE THREAD

**Issue by snoopy0815** [ocaml/Zarith#14]
unfortunately since [this code update](https://github.com/ocaml/Zarith/commit/96a8242def2023cf2d24cb29ec0a80437d5d4019) I'm not able anymore to install Zarith on Raspberry Debian Jessie :(

this is the error I get, after the command 'make':
```bash
pi@raspberrypi:~/.opam/system/build/zarith.1.6 $ make
gcc -DZ_OCAML_HASH -DZ_OCAML_COMPARE_EXT -DHAS_GMP     -c -o caml_z_arm.o caml_z_arm.S
caml_z_arm.S: Assembler messages:
caml_z_arm.S:25: Error: junk at end of line, first unrecognized character is `,'
<eingebaut>: die Regel für Ziel „caml_z_arm.o“ scheiterte
make: *** [caml_z_arm.o] Fehler 1

```
with earlier versions installation has been working without issues - and after commenting-out this line of code the problem vanishes as well...

do we really need this addition - or can we fix it in order to not break the installation on Debian Jessie?

Thanks for any hints / updates!
best regards.

------
[edited 11.10.2017 15:55]:
is related to https://github.com/ocaml/Zarith/pull/13

**Comment by bschommer** [comment]
Replacing the `@` with `%` in the `.section .note.GNU-stack,"",@progbits` should solve the problem.

**Comment by snoopy0815** [comment]
wow - this was fast, bschommer :)

and yes - perfectly right :+1:     this fixes the problem on Debian Jessie - wonderful!
Thank you very much!!

if this update finds its way into the next version - and [astrada/google-drive-ocamlfuse](https://github.com/astrada/google-drive-ocamlfuse) will link its installation scripts to this new version, the world will be perfect again :)
[sorry - of course this is off-topic - just for referencing where this problem arrises]

Thank you very much again!!
cheers.

**Comment by cfcs** [comment]
maybe worth considering including the compiler flag from here:
https://github.com/ocaml/Zarith/pull/13#issuecomment-331698256

**Comment by xavierleroy** [comment]
Fixed by pull request #15.
